Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/preferences/FlowPreference.kt

```diff
@@ -50,6 +51,19 @@ class FlowPreference<T> constructor(
     }
 
     companion object {
+        inline fun <reified T> jacksonReader(
```

Comment:
Add tests pls for reader & writer

Reply:
"reading and writing Jackson" test added